handbook/engineering/web/extensibility/index.md

```diff
@@ -11,41 +11,48 @@ The extensibility team owns our code host and third-party integrations (includin
   - [TJ Kandala](../../../../company/team/index.md#tharuntej-kandala-he-him)
   - [Murat Sutunc](../../../../company/team/index.md#murat-sutunc-he-him)  
 
-## Contact
+## Mission
 
-- [#extensibility-chat](https://app.slack.com/client/T02FSM7DL/C01LZKLRF0C) channel or @extensibility on Slack
-- On GitHub, mention or assign issues to the [@sourcegraph/extensibility](https://github.com/orgs/sourcegraph/teams/extensibility) team.
+Our mission is to bring the value of Sourcegraph to everywhere you work with code and to bring the value of other developer tools into Sourcegraph.
 
-## Goals
+## Vision
 
-TODO - link to Productboard roadmap
+Make Sourcegraph part of developers' daily workflow.
 
-## Ownership
+1. We'll build a healthy ecosystem of Sourcegraph extensions to bring the power of existing developer tools into our platform to empower developers.
+1. We'll augment existing developer workflows by building new ways to use Sourcegraph on other websites and IDEs.
 
-This team is responsible for:
+## Responsibilities
 
-1. Bringing the value of Sourcegraph to everywhere you work with code
-1. Bringing the value of other developer tools into Sourcegraph
+1. Sourcegraph extensions
+   - Providing a Sourcegraph extension API that enables developers to bring data from their favorite developer tools into their Sourcegraph workflow.
+   - Build and maintain useful Sourcegraph extensions.([Sourcegraph-maintained extensions](https://docs.sourcegraph.com/dev/background-information/sourcegraph_extensions)).
+   - Documentation and tutorials that enable third-party developers to create extensions that are actively used.
+   - Building and maintaining the extensions registry and discovery paths to create a community around Sourcegraph extensions.
+1. Code host integrations
+   - Add native support for Sourcegraph in code hosts (for example: GitLab) or popular sites where developers look at code examples (e.g. https://reactjs.org, https://pkg.go.dev/) so users don't have to install our browser extension to get code intelligence.
+1. Browser extensions
+   - Surface code intelligence and other Sourcegraph data in code hosts (for example: Github) through user-installed browser extensions.
```

Comment:
```suggestion
   - Develop and maintain [browser extensions](https://docs.sourcegraph.com/integration/browser_extension) that surface code intelligence and other Sourcegraph features on code hosts (for example: Github).
```
(It is technically possible, though rare, for someone to enable a chrome extension [for their whole workspace](https://support.google.com/chrome/a/answer/6306504?hl=en), I believe, so users don't "always" have to set it up.)